common/config.py

```diff
@@ -174,13 +181,16 @@ def __init__(self, config_path=None, data_path=None):
         else:
             self._LOCAL_CONFIG_PATH = os.path.abspath(config_path)
             self._LOCAL_CONFIG_FOLDER = os.path.dirname(self._LOCAL_CONFIG_PATH)
-        old_path = os.path.join(self._LOCAL_CONFIG_FOLDER, 'config2')
```

Comment:
Didn't I removed this in another PR?
Might be related to my messed up AboutDialog-PR where I assume to "solved" a merge conflict into the wrong direction.

Really sorry about that.

Reply:
Git is (a) be(a)st ;-)